src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -1224,28 +1239,26 @@ static String getRanges(JsonArray ar, String startProp, String endProp) {
    *   WHERE daterange($3, $4) @> lower(usageDateRange) AND openAccess
    *   GROUP BY titleEntryId
    * ) t1 ON t1.titleEntryId = title_entries.id
-   * WHERE agreementId = $1;
+   * WHERE agreementId = $1
+   *   AND NOT (printISSN IS NULL AND onlineISSN IS NULL)
    * </pre>
    */
-  private static void useOverTimeJournal(StringBuilder sql, TenantPgPool pool,
-      boolean openAccess, boolean unique, int periods) {
+  private static void useOverTime(StringBuilder sql, TenantPgPool pool,
+      boolean journal, boolean openAccess, boolean unique, int periods) {
 
-    sql.append("SELECT title_entries.kbTitleId AS kbId, kbTitleName AS title,"
-        + " printISSN, onlineISSN, ")
+    sql.append("SELECT title_entries.kbTitleId AS kbId, kbTitleName AS title,")
+        .append(journal ? " printISSN, onlineISSN, "
+                        : " ISBN, ")
         .append(openAccess ? "'OA_Gold' AS accessType, "
                            : "'Controlled' AS accessType, ")
         .append(unique ? "'Unique_Item_Requests' AS metricType, "
                        : "'Total_Item_Requests' AS metricType, ");
     for (int i = 0; i < periods; i++) {
       if (i > 0) {
-        sql.append(" + ");
+        sql.append(", ");
```

Comment:
Is this fixing a straight-up mistake?

Reply:
No, it is a misleading diff created by GitHub.
This column has been removed, see example in (old) line 1211:
```n0 + n1 AS total,```

Reply:
OK.